### PR TITLE
[*] CORE: New parameter for Tools::fileAttachment

### PR DESCRIPTION
**Pull request by gskema**
- useful method for the developers, however, sometimes we don't want to grab the file contents and hold it in memory (when the file is large - 10MB, 100MB ...)

### CODE REVIEW

File: classes/Tools.php [hunk 1]
```diff
@@ -3264,19 +3264,30 @@ public static function addonsRequest($request, $params = array())
...
+	/**
+	 * Returns an array containing information about
+	 * HTTP file upload variable ($_FILES)
+	 *
+	 * @param string $input          File upload field name
+	 * @param bool   $return_content If true, returns uploaded file contents
+	 *
+	 * @return array|null
+	 */
+	public static function fileAttachment($input = 'fileUpload', $return_content = true)
```
Comment (jeromenadaud) [inline classes/Tools.php]: Hi, please set $return_content to false by default in order to keep original logic.

Best regards

Comment (gskema) [inline classes/Tools.php]: But it already **is** organic ?

Comment (jeromenadaud) [inline classes/Tools.php]: Sorry, yes github is very horrible sometimes for checking PR.

**Comment by jeromenadaud** [comment]
Hi,

Thank you for your time and dedication.

Best regards.

**Pull request merged**